src/modular/bingcd/extension.rs

```diff
@@ -1,11 +1,20 @@
-use crate::{ConstChoice, ConstCtOption, Int, Limb, Uint};
+use crate::{ConstChoice, ConstCtOption, Limb, Uint};
 
 pub(crate) struct ExtendedUint<const LIMBS: usize, const EXTENSION_LIMBS: usize>(
     Uint<LIMBS>,
     Uint<EXTENSION_LIMBS>,
 );
 
 impl<const LIMBS: usize, const EXTRA: usize> ExtendedUint<LIMBS, EXTRA> {
+    /// Construct an [ExtendedUint] from the product of a [Uint<LIMBS>] and an [Uint<EXTRA>].
```

Comment:
Using this syntax will linkify the resulting rustdoc:

```suggestion
    /// Construct an [`ExtendedUint`] from the product of a [`Uint<LIMBS>`] and an [`Uint<EXTRA>`].
```

Reply:
Going to approve this PR but it would be good to get any of these which are visible in public rustdoc fixed up

Reply:
(Also, on my end `[Uint<LIMBS>]` links to `Uint` and gets displayed as `Uint`, while
```
[`Uint<LIMBS>`]
```
still links to `Uint` but gets formatted as `Uint<LIMBS>`.

So, the backticks just add the `LIMBS`, but not the link.)